lib/util/git/index.ts

```diff
@@ -669,9 +671,13 @@ export async function commitFiles({
         }
       }
     }
-    const commitRes = await git.commit(message, [], {
-      '--no-verify': null,
-    });
+
+    const commitOptions = {};
```

Comment:
Is there an interface for this? if not declare as `Record<string, unknown>`

Reply:
Just checked and they do have a type for it. It's just a generic `Record<string, string | number | null>` type thing, but might as well import it and use it in case they ever improve/change their typing.